cmd/system-probe/config/adjust_usm.go

```diff
@@ -38,8 +38,11 @@ func adjustUSM(cfg config.Config) {
 	applyDefault(cfg, smNS("http_idle_connection_ttl_in_s"), 30)
 	deprecateInt64(cfg, netNS("http_notification_threshold"), smNS("http_notification_threshold"))
 	applyDefault(cfg, smNS("http_notification_threshold"), 512)
+	// http_max_request_fragment
```

Comment:
This comment doesn't seem to be of value, since it is just the string value used on the next two lines.